draft-ietf-pquip-pqc-engineers.md

```diff
@@ -260,6 +260,7 @@ informative:
 --- abstract
 
 The advent of a cryptographically relevant quantum computer (CRQC) would render state-of-the-art, traditional public-key algorithms deployed today obsolete, as the mathematical assumptions underpinning their security would no longer hold. To address this, protocols and infrastructure must transition to post-quantum algorithms, which are designed to resist both traditional and quantum attacks. This document explains why engineers need to be aware of and understand post-quantum cryptography (PQC), detailing the impact of CRQCs on existing systems and the challenges involved in transitioning to post-quantum algorithms. Unlike previous cryptographic updates, this shift may require significant protocol redesign due to the unique properties of post-quantum algorithms.
+The authors would like to emphasize the fact that words/phrases like "current", "state-of-the art", or "ongoing" denotes the work/research/investigations/deployments/developments happening at the time of publishing this document.
```

Comment:
I would re-phrase it as follows:
The terms "current," "state-of-the-art," and "ongoing," as used in this document, refer to work, research, investigations, deployments, or developments that are applicable at the time of publication.

Please add above line to Introduction and remove it from the Abstract.